.circleci/config.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+jobs:
+  build:
+    branches:
+      ignore:
+        - master # only test new branches
+    docker:
+      - image: circleci/node:12
```

Comment:
why not github actions?

Reply:
I think circleci has nicer formatting of linting/test results. 

Added a github workflow to this branch so we can try it out. It's almost the same as the circleci config but I found no quick solution on how to manage junit formatted test results

Reply:
Ok I thought it's gonna show the action as soon as I have the workflow file. Won't spend any more 'unplanned' time on this.

Do you have experience with github actions?